Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -1203,6 +1203,11 @@
       "unitless-length"
     ]
   },
+  "max-inline-size": {
+    "logical-alias-for": "max-width",
+    "__comment": "Also a logical alias for max-height",
```

Comment:
Can we make `logical-alias-for` an array instead?
`max-width` and `min-width` accept slightly different values, and `max-inline-size` needs to accept the union of all those values.

Reply:
Oh wait, what I'm saying doesn't make any sense 😅 I'm mixing up min and max..